assistants/orderAssistant.js

```diff
@@ -1,22 +1,32 @@
-import {PAGES} from "../pages/pages";
-import {Assistants} from "./assistants";
-import {expect} from "@playwright/test";
-
+import { PAGES } from "../pages/pages";
+import { USERS } from "../models/users";
 
 export class OrderAssistant {
+  async placeTheOrder(page) {
```

Comment:
```suggestion
  async placeTheOrder(page) {
```

you should put user object as parameter to this method. And use user object inside method.
      -> 
```  async placeTheOrder(page, userObj) {
                    await page.locator(PAGES.orderPopup.city).fill(userObj.userCity);
        }
```